recipes/frecentf

```diff
@@ -0,0 +1,3 @@
+(frecentf :fetcher git
+	  :url "https://git.launchpad.net/frecentf.el"
+	  :commit "origin/master")
```

Comment:
I think we should remove this line.  This looks like a branch name, and even then, MELPA will take `master` by default.

Reply:
I guess you meant the "commit" property (github highlights the three lines)

Reply:
Yep!  Thanks.